driver-core/src/main/com/mongodb/event/ConnectionEventMulticaster.java

```diff
@@ -16,63 +16,73 @@
 
 package com.mongodb.event;
 
-import com.mongodb.annotations.Beta;
+import com.mongodb.annotations.Immutable;
+import com.mongodb.diagnostics.logging.Logger;
+import com.mongodb.diagnostics.logging.Loggers;
 
-import java.util.Set;
-import java.util.concurrent.ConcurrentHashMap;
+import java.util.ArrayList;
+import java.util.List;
 
-import static java.util.Collections.newSetFromMap;
+import static com.mongodb.assertions.Assertions.isTrue;
+import static com.mongodb.assertions.Assertions.notNull;
+import static java.lang.String.format;
+import static java.util.Collections.unmodifiableList;
 
 /**
- * A multicaster for connection events.
+ * A multicaster for connection events. Any exception thrown by one of the listeners will be caught and not re-thrown, but may be
+ * logged.
+ *
+ * @since 3.5
```

Comment:
As we've changed it I marked it to be `@since 3.5` not sure if thats correct or not.